.github/contributing.md

```diff
@@ -62,6 +62,10 @@ Open your web browser to [localhost:4000](http://localhost:4000/).
 - Wait for feedback, or for your branch to be merged!
 - We typically delete branches after the code has been merged to master, so feel free to make a new branch for each new issue you work on.
 
+## Updating page content
+Content can be updated via the markdown files in the `src` folder. Some content is generated by adding html includes to the markdown files (e.g. `{% include steering-committee.html %}` within the `about-us.md` file). See the `src` > `_includes` folder.
+
```

Comment:
I think it would be helpful to add a note about the difference between using markdown over HTML. My understanding is that Markdown files are preferred, and intended to be more approachable to edit for the less tech-savvy. However, some layouts are more complex, like the steering committee partial you mentioned, and necessitate using HTML.

Reply:
Good suggestion! This appears to be resolved in the latest edits.